### PR TITLE
 test: add unit tests for dispatch-series helper functions

### PR DESCRIPTION
**Pull request by JSweet314**
## Summary
- Add unit tests for all exported functions in `src/core/dispatch-series.ts`
- Cover `requireIntInRange`, `clampIosSwipeDuration`, `shouldUseIosTapSeries`, `shouldUseIosDragSeries`,
 `computeDeterministicJitter`, and `runRepeatedSeries`
- 29 tests total covering boundary values, error cases, and behavioral expectations

## Validation
- `pnpm node --test src/core/__tests__/dispatch-series.test.ts` → 29/29 pass

```
ℹ tests 29
ℹ suites 0
ℹ pass 29
ℹ fail 0
ℹ cancelled 0
ℹ skipped 0
ℹ todo 0
ℹ duration_ms 234.259217
```

### CODE REVIEW

File: src/core/__tests__/dispatch-series.test.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+});
+
+test('runRepeatedSeries pauses between operations but not after the last', async () => {
+  const pauseMs = 50;
+  const start = Date.now();
+  await runRepeatedSeries(3, pauseMs, async () => {});
+  const elapsed = Date.now() - start;
+  // 3 operations with pauses only between them = 2 pauses
+  assert.ok(elapsed >= pauseMs * 2 - 10, `elapsed ${elapsed}ms should be >= ${pauseMs * 2 - 10}ms`);
+  assert.ok(elapsed < pauseMs * 3 + 50, `elapsed ${elapsed}ms should be < ${pauseMs * 3 + 50}ms`);
```
[the remark this answers is not in the document]
Comment (JSweet314) [reply src/core/__tests__/dispatch-series.test.ts]: Replaced the wall-clock timing with `t.mock.method(globalThis, 'setTimeout', ...)` which mocks setTimeout to invoke callbacks immediately and records the delay values. The test now asserts both operation order ([0, 1, 2]) and that exactly 2 pauses were scheduled ([50, 50]), confirming no pause after the last operation. No real time passes, so no flakiness under CI load.

**Comment by thymikee** [comment]
some refactorings should be good first issues :)
